web-client/src/ducks/requests/functions.ts

```diff
@@ -51,39 +51,39 @@ export const setUserRequest = async ({
     .withConverter(RequestFirestoreConverter)
     .set(requestPayload);
 
-export const getOpenRequest = async ({ lat, lng }: IgetOpenRequests) =>
+export const getOpenPost = async ({ lat, lng }: IgetRequestPosts) =>
   functions.httpsCallable('https-api-requests-getRequests')({
```

Comment:
can you make use of firebase directly here to get the data rather than the cloud function? that is what we are aiming for, so we shouldn't use the cloud function here.

Reply:
Yes, I am working on that, but would like it to be a separate pull request.